api-specificatie/openapi.yaml

```diff
@@ -866,6 +866,8 @@ components:
     AdresHalCollectie:
       type: object
       properties:
+        _links:
+          $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/schemas/HalCollectionLinks'
```

Comment:
Ik mis hier een self link.

Reply:
De self link zit in de HalCollectionLinks

Reply:
Ook hier zit de selflink in de component waarnaar gerefereerd word.